interpreter/cling/tools/packaging/cpt.py

```diff
@@ -1869,7 +1871,7 @@ def custom_input(prompt, always_yes=False):
     workdir = os.path.abspath(os.path.expanduser(args['with_workdir']))
 srcdir = os.path.join(workdir, 'cling-src')
 CLING_SRC_DIR = os.path.join(srcdir, 'tools', 'cling')
-CPT_SRC_DIR = os.path.join(CLING_SRC_DIR, 'tools', 'packaging')
+# CPT_SRC_DIR = os.path.join(CLING_SRC_DIR, 'tools', 'packaging')
```

Comment:
Can you remove the commented code? This PR seems good to go.

Reply:
Hi @vgvassilev just removed the commented code